tests/test_calibration_format.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from src.calibrationFormatProcessor import CalibrationFormatProcessor
+from src.calibrationFormat import Unit, CalibrationFormat
+from pydantic import ValidationError
+
+def test_calibration_read():
```

Comment:
Can I tempt you to look at the [bit about tests naming in the dev standards doc](https://imperialcollege.atlassian.net/wiki/spaces/PMLSD/pages/429555802/Developer+Standards) - would be good to try and get everyone being consistent at all possible.

> Consider test name structure [function_or_feature_under_test]_does_[some_single_action]_when_[this_input_is_satisfied] 
> e.g. generate_reports_creates_plot_of_absolute_field_when_at_least_24h_of_science_data_is_available.